portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -62,6 +62,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String commentText = getParameter(request, "commentText", "");
     long timestamp = System.currentTimeMillis();
 
+    if (commentText.replaceAll("\\s+", "").equals("")) {
```

Comment:
Add a comment about what are you trying to do here. Not the most readable code.

Reply:
I would even go as far as making the regex string a variable something like WHITESPACES_REGEXP = "\\s+"

This might also be more readable on two lines of code:
commentText.replaceAll("\\s+", "")
if commentText.equal("") {}

Reply:
That sounds fine.

Reply:
Fixed this 👍 thanks for the feedback